include/boost/fusion/container/vector/vector.hpp

```diff
@@ -306,13 +295,11 @@ namespace boost { namespace fusion
 
         template <
             typename Sequence
-          , typename Sequence_ = typename remove_reference<Sequence>::type
-          , typename = typename boost::enable_if_c<(
-                !is_base_of<vector, Sequence_>::value &&
```

Comment:
This check was added to speedup the compile performance of copies/moves. This constructor is now used for copies/moves, and has to instantiate an `at_c<...>(...)` for each element instead of using the compiler generated default. I didn't add a comment or store this information anywhere, so maybe am I mistaken.

I will try to investigate this again and update with a comment + PR. I commented in this PR in case I forget to do it, someone else might want to know.

Reply:
If so, using `is_same` is better than `is_base_of`.